README.md

```diff
@@ -106,11 +106,20 @@ The tests are all located in the `tests` subdirectory of the project.
  * Old profiles associated with each test are JIT removed
  * New profiles are moved to subdirectories indicating which test they are assosciated with
 
-A limitation of the tests is that they assume a TAU installation matching the [salt-dev]
+By default the tests assume a TAU installation matching the [salt-dev]
 development image.
 If you don't have a GCC and Clang configuration of TAU installed into the default location
 (`/usr/local/x86_64/`) then some of the tests will fail.
 
+Alternatively you can add `-DTAU_ROOT=<dir>` to the cmake invocation to point the tests
+to a different location. The tests require TAU to be built with these two configurations:
```

Comment:
So the tests pass when setting -DTAU_ROOT? Perhaps we should update the paragraph above too. Right not is reads like:

>This doesn't work. Ok, or you can try this...

I.e. a bit self contradictory. I can help with wording.